src/apiHandlers/beneficiary.ts

```diff
@@ -0,0 +1,20 @@
+import beneficiaries from '../data/beneficiaries';
+
+export const getBeneficiaries = async () =>
+    // try {
+    //     const res = await axios.get(`${process.env.NEXT_PUBLIC_API}/beneficiary`);
+    //     const beneficiaries = res.data;
+    //     return { success: true, beneficiaries };
+    // } catch (err) {
+    //     return { success: false, beneficiaries: [] };
+    // }
+
+    ({ success: true, beneficiaries });
+
+export const getSingleBeneficiary = async (beneficiaryId: string) => {
```

Comment:
Is it not being used? Then you can remove it.
If in use, return the beneficiary from the list or null when it does not exist.